src/Common/src/CoreLib/System/Numerics/Vector.cs

```diff
@@ -2747,7 +2683,7 @@ public string ToString(string format, IFormatProvider formatProvider)
         [MethodImplAttribute(MethodImplOptions.AggressiveInlining)]
         public static Vector<T> operator ~(Vector<T> value)
         {
-            return s_allOnes ^ value;
+            return allOnes ^ value;
```

Comment:
Does this compile?

Reply:
Probably not, having issues with building on my machine so I'm leaning on your build server.